packages/cli/README.md

```diff
@@ -127,3 +127,86 @@ jobs:
       - run: npm ci
       - run: npx code-pushup autorun --upload.apiKey=${{ secrets.PORTAL_API_KEY }}
 ```
+
+# CLI
+
+## Global Options
+
+| Option           | Type      | Default                 | Description                                                            |
+| ---------------- | --------- | ----------------------- | ---------------------------------------------------------------------- |
+| **`--progress`** | `boolean` | `true`                  | Show progress bar in stdout.                                           |
+| **`--verbose`**  | `boolean` | `false`                 | When true creates more verbose output. This is helpful when debugging. |
+| **`--config`**   | `string`  | `code-pushup.config.js` | Path to the config file, e.g. code-pushup.config.js                    |
+
+## Commands
+
+### `collect` command
+
+Run commands over:
+`code-pushup collect [options]`
+
+Description:  
+Run Plugins and collect results.
+
+| Option                    | Type       | Default | Description                                     |
+| ------------------------- | ---------- | ------- | ----------------------------------------------- |
+| **`--onlyPlugins`**       | `string[]` | `[]`    | Only run the specified plugins.                 |
+| **`--persist.outputDir`** | `string`   | n/a     | Directory for the produced reports.             |
```

Comment:
Maybe out of scope of this PR, but why don't we specify a default location for `outputDir`? E.g. `'.code-pushup'`.

Reply:
Also thought about it!

Reply:
@matejchalk created #321